### ISSUE TITLE
Install fails on node v10

### ISSUE THREAD

**Issue by ptz0n** [agnat/node_mdns#215]
Installing & testing `node-gyp` separately works just fine.

```shell
$ node --version && npm --version
v10.0.0
6.0.0
```

```shell
$ npm install

> mdns@2.3.4 install /Users/eng/Code/node_mdns
> node-gyp rebuild

  CXX(target) Release/obj.target/dns_sd_bindings/src/dns_sd.o
In file included from ../src/dns_sd.cpp:1:
In file included from ../src/mdns.hpp:12:
In file included from ../node_modules/nan/nan.h:190:
../node_modules/nan/nan_maybe_43_inl.h:88:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(GetCurrentContext(), key, value, attribs);
         ~~~  ^
…
```

__Full debug log:__ https://gist.github.com/ptz0n/a209c2ce76ef8131253696320e306018

**Comment by ylor** [comment]
+1 also seeing this

**Comment by kode54** [comment]
Affecting the "nan" module, which I see is using an interface that was deprecated in 2015:

https://github.com/nodejs/nan/issues/504

**Comment by agnat** [comment]
Fixed in v2.4.0: Node v10 should work fine now.
